cmd/ebitenmobile/_files/EbitenView.java

```diff
@@ -329,11 +363,19 @@ private static int getAxisMask(InputDevice joystickDevice) {
         final int SDL_CONTROLLER_AXIS_TRIGGERRIGHT = 5;
 
         int naxes = 0;
+        boolean have_z = false;
+        boolean have_past_z_before_rz = false;
```

Comment:
Can we use `fooBar` instead of `foo_bar` for consistency?

Reply:
Done.